features/FEATURE_LWIP/lwip-interface/ppp_lwip.cpp

```diff
@@ -18,6 +18,14 @@
 #include "platform/FileHandle.h"
 #include "platform/mbed_poll.h"
 #include "events/EventQueue.h"
+#if defined(FEATURE_COMMON_PAL)
+#include "mbed_trace.h"
+#define TRACE_GROUP "LPPP"
+#else
+#define tr_debug(...) (void(0)) //dummies if feature common pal is not added
```

Comment:
> What do you mean by "Conditional FEATURE_COMMON_PAL is not acceptable" ?
Its just a macro test, which tells me if I can use mbed-trace or not.

I understand is not scope of this PR, but mbed-trace should become own library and not use "if some feature that includes this module". Thus it invokes a feeling this library is not yet ready , we will need to make it generic and use it widely, currently it is not as I see it.

@sg- What do you think?